code/modules/antagonists/pirate/pirate_outfits.dm

```diff
@@ -68,3 +68,36 @@
 	head = /obj/item/clothing/head/costume/crown
 	mask = /obj/item/clothing/mask/cigarette/cigar/havana
 	l_pocket = /obj/item/lighter
+
+/datum/outfit/pirate/interdyne
+	name = "Ex-Interdyne Pharmacist"
+
+	id = /obj/item/card/id/advanced/black
+	id_trim = /datum/id_trim/job/medical_doctor
+	uniform = /obj/item/clothing/under/rank/medical/scrubs/coroner
+	suit = /obj/item/clothing/suit/toggle/labcoat
+	back = /obj/item/storage/backpack/satchel/med
+	glasses = /obj/item/clothing/glasses/hud/health/night
+	gloves = /obj/item/clothing/gloves/color/black
+	head = /obj/item/clothing/head/utility/surgerycap/black
+	shoes = /obj/item/clothing/shoes/sneakers/white
+	l_pocket = /obj/item/card/emag/doorjack
+
+/datum/outfit/pirate/interdyne/captain
+	name = "Ex-Interdyne Senior Resident"
+
+	id_trim = /datum/id_trim/job/chief_medical_officer
```

Comment:
Should these really be using station trims and not custom pirate trims?

Reply:
I can make them syndicate type althought I like them to look medical type for sec huds and ghost huds

Reply:
> I can make them syndicate type althought I like them to look medical type for sec huds and ghost huds

ID hud icons don't work off ID name anymore, so that's not a concern if you figure out how to add new interdyne job datums. It'd be cool if Interdyne id cards had the + icon, but with syndicate red coloring

Reply:
Should it be a cross like for nanotrsen or have a different logo